### DISCUSSION TITLE
[Question] 3D game mode

### DISCUSSION THREAD

**Discussion by MarvNC** [Blinue/Magpie#500] [General]
What exactly does this option do?
![image](https://user-images.githubusercontent.com/17340496/225815954-4cb98294-b1ba-4b9c-b873-f83452b865a5.png)

**Comment by MarvNC** [comment]
Never mind, I found out that it just restricts your cursor.

**Reply by Blinue** [reply]
https://github.com/Blinue/Magpie/discussions/484#discussioncomment-5333533
